### ISSUE TITLE
Use with skypack

### ISSUE THREAD

**Issue by LeHibou2** [marcofugaro/three-projected-material#29]
Using this lib with skypack leads to a black screen:

```
        <script type="module">

                import {
                  ...
                } from "https://cdn.skypack.dev/three@0.136.0";

                import threeProjectedMaterial from 'https://cdn.skypack.dev/three-projected-material@v2.0.5/';
                 ...


                        const geometry = new PlaneGeometry(10, 10);
                        const material = new threeProjectedMaterial({
                                camera: camera, // the camera that acts as a projector
                                texture: get_texture("texture"), // the texture being projected
                                textureScale: 0.8, // scale down the texture a bit
                                textureOffset: new Vector2(0.1, 0.1), // you can translate the texture if you want
                                cover: true, // enable background-size: cover behaviour, by default it's like background-size: contain
                                color: '#ccc', // the color of the object if it's not projected on
                                roughness: 0.3, // you can pass any other option that belongs to MeshPhysicalMaterial
                        })
                        const plane = new Mesh(geometry, material);
                        plane.position.set(-20, 0, 0)
                        scene.add(plane);
        </script>
```

Any idea what is going on?
Note: camera and texture have to be expressed with object syntax 

Thank you

**Comment by marcofugaro** [comment]
Look at the basic example: https://github.com/marcofugaro/three-projected-material/blob/5e0ed6d0be30db43850384fb09acb7400a2d718f/examples/basic.html#L23

if you replace 

```diff
-import ProjectedMaterial from './lib/ProjectedMaterial.module.js'
+import ProjectedMaterial from 'https://cdn.skypack.dev/three-projected-material@v2.0.5'
```

everything works as expected.

**Comment by LeHibou2** [comment]
Thanks, but I still have:
` material.onBeforeRender is not a function error`

Strange..

**Comment by marcofugaro** [comment]
Could you post your full code? I can't debug that

**Comment by marcofugaro** [comment]
Fixed in [v2.1.0](https://github.com/marcofugaro/three-projected-material/releases/tag/v2.1.0), your example should work fine now.
